data/store/mongo/mongo_datum.go

```diff
@@ -73,13 +73,16 @@ func (d *DatumRepository) EnsureIndexes() error {
 		},
 		{
 			Keys: bson.D{
+				{Key: "_userId", Value: 1},
```

Comment:
Did you mean to include these Mongo changes in this PR?

Reply:
Did this get picked up accidentally due to a merge from master?

Reply:
ahh crap - no i didn't